app/src/main/AndroidManifest.xml

```diff
@@ -22,7 +22,6 @@
 
     <application
         android:name=".ApplicationMain"
-        android:allowBackup="false"
```

Comment:
If you do this, you must clearly indicate what is saved or not. Because in case of restoration it can generate bugs